### PR TITLE
[release-1.1][CPS-4825] feat: support concat multiline log

### PR DESCRIPTION
**Pull request by jianglinjian**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglinjian** [comment]
/hold
待调试

File: build/filebeat/filebeat.yml.tpl [hunk 1]
```diff
@@ -19,6 +19,27 @@ filebeat.inputs:
     cluster: ${CLUSTER_ID}
     node_name: ${NODE_NAME}
     component: system.docker
+    {{- range $k,$v := .fields }}
+    {{ $k }}: {{ $v }}
+    {{- end }}
+  {{ if .multiline -}}
+  multiline:
```
Comment (lichuan0620) [inline build/filebeat/filebeat.yml.tpl]: multiline 给容器的配置里加上就行了，docker 和 kubelet 的不需要

Comment (jianglinjian) [reply build/filebeat/filebeat.yml.tpl]: fixed

**Comment by jianglinjian** [comment]
@lichuan0620 PTAL

**Pull request merged**